backend/pom.xml

```diff
@@ -71,6 +71,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-cache</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
```

Comment:
Is(n't) that pulled via the quarkus-cache dep in the entry above?

Reply:
It should and the version has to come from the Quarkus BOM anyway.